internal/setup/setup.go

```diff
@@ -535,8 +536,7 @@ func parsePackage(baseDir, pkgName, pkgPath string, data []byte) (*Package, erro
 				Mutate: yamlSlice.Mutate,
 			},
 		}
-
-		for _, refName := range yamlSlice.Essential {
+		for _, refName := range append(yamlSlice.Essential, yamlPkg.Essential...) {
```

Comment:
It feels a bit more natural to have this inverted, so that the package-level essentials come first.

Reply:
Per your other comment https://github.com/canonical/chisel/pull/127#discussion_r1562685404, this bit of code has now been split into two different loops.